front/package.json

```diff
@@ -33,6 +33,7 @@
     "@radix-ui/react-scroll-area": "^1.0.5",
     "@radix-ui/react-select": "^2.0.0",
     "@radix-ui/react-slot": "^1.0.2",
+    "@radix-ui/react-visually-hidden": "^1.1.2",
```

Comment:
Adding this component as some `radix` components requires to render to some inner component. See [docs](https://www.radix-ui.com/primitives/docs/utilities/visually-hidden)